gradle/web-gwt-test.gradle

```diff
@@ -21,7 +21,7 @@ task 'gwtTest', type: Test, {
                                'dhTestServer': testServer
         ]
         t.include '**/*TestSuite.class'
-        t.logging.level = 'INFO'
+//        t.logging.level = 'INFO'
```

Comment:
Is this intended for merge? Probably needs to be uncommented or deleted.

Reply:
Ah yes, I forgot, this doesn't gradle-ism doesn't work anymore. Will need to see how to fix it...

Reply:
I removed this line from James' advice. I think it was used for manually inspection when things went wrong. If things go wrong gwtTest, we can look into `t.testLogging { ... }` to re-enable (I wasn't able to figure it out).